generators/server/templates/src/main/java/package/service/_UserService.java

```diff
@@ -274,7 +274,7 @@ public User createUser(UserDTO userDTO) {
      <%_ } _%>
      */
     public void updateUser(String firstName, String lastName, String email, String langKey<% if (databaseType === 'mongodb' || databaseType === 'sql') { %>, String imageUrl<% } %>) {
-        userRepository.findOneByLogin(SecurityUtils.getCurrentUserLogin()).ifPresent(user -> {
+        userRepository.findOneByLogin(SecurityUtils.getCurrentUserLogin().orElse(null)).ifPresent(user -> {
```

Comment:
A bit simpler: 

`SecurityUtils.getCurrentUserLogin().map(userRepository::findOneByLogin).ifPresent( .... )`

And the userRepository.findOneByLogin wont be called with null.

Reply:
You also need a map(Optional::get) step. So in the end it's not simpler

Reply:
Where do we need map(Optional::get) ?
This would needed, only with Optional<Optional<String>>

Reply:
That's what you get after the first map

Reply:
we can .flatMap() instead of .map()

Reply:
Sorry, flatMap !

Reply:
Yes flatMap should work. I'll try it tomorrow